lib/execution_engine2/sdk/EE2Runjob.py

```diff
@@ -460,6 +504,130 @@ def _check_job_arguments(self, jobs, has_parent_job=False):
             # although most likely jobs aren't operating on the same object
             self._check_ws_objects(source_objects=job.get(_SOURCE_WS_OBJECTS))
 
+    def retry_multiple(self, job_ids, as_admin=False):
+        """
+        #TODO Add new job requirements/cgroups as an optional param
+        :param job_ids: The list of jobs to retry
+        :param as_admin: Run with admin permission
+        :return: The child job ids that have been retried or errors
+        """
+        retried_jobs = []
+        for job_id in job_ids:
+            try:
+                child_job_id = self.retry(job_id, as_admin=as_admin)
+                retried_jobs.append({"job_id": child_job_id})
+            except Exception as e:
+                retried_jobs.append({"job_id": None, "error": f"{e}"})
+        return retried_jobs
+
+    def retry(self, job_id, as_admin=False) -> str:
+        """
+        #TODO Add new job requirements/cgroups as an optional param
+        :param job_id: The main job to retry
+        :param as_admin: Run with admin permission
+        :return: The child job id that has been retried
+        """
+        job = self.sdkmr.get_job_with_permission(
+            job_id, JobPermissions.WRITE, as_admin=as_admin
+        )  # type: Job
+
+        # Cannot retry a retried job, you must retry the parent
+        if job.retry_parent:
+            raise CannotRetryARetryException(
+                f"Cannot retry a retried job attempt, please retry the parent {job.retry_parent}"
```

Comment:
Assuming that `job.retry_parent` is the ID of the job that the user should have entered, if you're emitting this message you may as well have ee2 do the substitution and rerun the job...

Reply:
Great idea, done!

Reply:
Hmm, but that would make the parent job get canceled...

Reply:
And it would kill the parent's jobs..